Shows all related documents in `details`.

gkellogg opened the pull request:
(Note, build will complain that it is out of sync with the rdf-common repository, until we make it a global change).

See rendered version [here](https://raw.githack.com/w3c/sparql-entailment/related-details/spec/index.html).


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/sparql-entailment/pull/31.html" title="Last updated on Jun 15, 2023, 9:06 PM UTC (ff62487)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/sparql-entailment/31/9c8cc3f...ff62487.html" title="Last updated on Jun 15, 2023, 9:06 PM UTC (ff62487)">Diff</a>

afs commented:
Thank you for providing the rendered link.

It says 10 and 12 recommendations. 

RDF is 9 recs, 2 notes (New, UCR - assuming UCR docs are usually notes)
SPARQL is 11+1

Better as 11 and 12 documents.
Or not mention the numbers, and say "a set of documents".

pfps reviewed (commented):
As Andy says, plus UCR hasn't even been approved as a note.

gkellogg commented:
RDF12-UCR was left in accidentally. Note that the GitHack view may take some time to show the adjusted set of documents due to server-side caching.

TallTed commented:
1. `10. SPARQL 1.2 Entailment Regimes(this document)` should have a space preceding the open-paren.

2. For consistency (as well as easy accuracy when/if we decide we'll publish the UCR, and add it to the RDF 1.2 list), I think `RDF 1.2 Recommendations:` and `SPARQL 1.2 Recommendations:` should both be `Documents` instead of `Recommendations`

gkellogg commented:
I had changed "recommendations" to "documents" in one place, but missed another.

The space before "(this document)" is done in the fixup.js script, now fixed.

pfps reviewed (approved):
Looks good now.

gkellogg commented:
See https://github.com/w3c/rdf-common/pull/7; once that's merged, this will also be ready to merge.
